keymap.cson

```diff
@@ -23,3 +23,14 @@
 # This file uses CoffeeScript Object Notation (CSON).
 # If you are unfamiliar with CSON, you can read more about it here:
 # https://github.com/bevry/cson#what-is-cson
+
+# window/pane navigation
+'body, atom-workspace atom-text-editor:not([mini])':
```

Comment:
I presume you need to include that if you are not in an editor window and want to navigate, like when you are in settings or find results, etc. I haven't tested that though.